README.md

```diff
@@ -10,12 +10,19 @@ Let's use the apple's official api design guidelines https://swift.org/documenta
 - 4 spaces, not tabs (Text Editing > Indentation)
 - use `swiftlint` https://github.com/realm/SwiftLint
 
+## SonarQube Style Guidelines
+
+Don't forget about SonarQube properties. We should add to exclusions:
+sonar.exclusions=Artifacts/\*\*,Application/ProjectName/Assets.xcassets/\*\*,Application/Pods/\*\*
+
 ## Best Practices
 
 Let's use swift community best practices https://github.com/schwa/Swift-Community-Best-Practices
 
 Also take a look at our [commit style](commit-style.md)
 
+Don't save commented code in your files. You can use Version Editor in Xcode to see the file's history.
+
```

Comment:
I believe this is mentioned somewhere in guides, but it's nothing wrong to mention it one more time )